### ISSUE TITLE
Relation many selector visibility problem

### ISSUE THREAD

**Issue by ablunier** [oscarotero/folk#2]
When you put a relation many field as last element in a form, the dropdown selector is not displayed:

![seleccion_003](https://cloud.githubusercontent.com/assets/4113587/24669412/acd9aabc-196a-11e7-871f-80979075a057.png)

**Comment by oscarotero** [comment]
Yes, I'm aware of that, but not sure what is the best way to fix it. It's due the `position: sticky` of the buttons bar, and as long as I know, it cannot be solved with `z-index`.

I've planned to change to css grid in next releases, so maybe it fixes that.

Meanwhile, currently I'm avoiding putting these fields at the end 😄

**Comment by ablunier** [comment]
Ok, no problem then! I first tried to fix it with ```z-index``` but wasn't able to make it. I reported it just in case that you were not aware of it.

I came up to the same conclusion as you: don't put these fields at the end :smile: :+1:
